source/PlayState.hx

```diff
@@ -1,23 +1,64 @@
 package;
 
-import flixel.FlxSprite;
+import Wall;
+import flixel.FlxBasic;
+import flixel.FlxG;
+import flixel.FlxObject;
 import flixel.FlxState;
-import hero.Hero;
+import flixel.group.FlxGroup.FlxTypedGroup;
+import flixel.text.FlxText;
+import hero.ControllableHero;
+import hero.ScreensaverHero;
 
 class PlayState extends FlxState
 {
-	var hero:Hero;
+	var controllableHero:ControllableHero;
+	var walls:FlxTypedGroup<Wall>;
+	var screensaverHeroes:FlxTypedGroup<ScreensaverHero>;
+	var totalScore:Int = 0;
 
 	override public function create()
 	{
 		super.create();
 
-		hero = new Hero(300, 300);
-		add(hero);
+		screensaverHeroes = new FlxTypedGroup<ScreensaverHero>();
+		for (i in 0...30)
+		{
+			screensaverHeroes.add(new ScreensaverHero(300, 50));
+		}
+		add(screensaverHeroes);
+
+		var controllableHero = new ControllableHero();
```

Comment:
By putting `var` here in front of `controllableHero` we're making this a local variable instead of populating our instance variable `controllerHero` defined on line 15.

This probably isn't desirable - we want `controllableHero` populated at the instance level so we can reuse it later during `update()`